browser/extensions/api/brave_rewards_api.cc

```diff
@@ -636,6 +642,10 @@ void BraveRewardsAttestPromotionFunction::OnAttestPromotion(
   Respond(TwoArguments(base::Value(static_cast<int>(result)), std::move(data)));
 }
 
+BraveRewardsGetPendingContributionsTotalFunction::
+    BraveRewardsGetPendingContributionsTotalFunction()
+    : weak_factory_(this) {}
```

Comment:
functions are refcounted, so I gather we don't need weak pointers for them

Reply:
Good point. I'll remove this and also update the other pre-existing cases in that file.